examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
+
+(class extends HTMLElement {
+
+  initialize () {
+    this.context.current = 0
+
+    this.context.questions_and_answers =
+      [
+        {
+          question: 'what popular tool for developing the client side application should we use?',
+          answers: ['React', 'Ember.js', 'Vue', 'Angular', 'Bootstrap', 'jQuery'],
+          // answer: undefined // There is no answer hehe
+        },
+
+        {
+          question: 'What city is known as "The Big 🍎 Apple ?"',
+          answers: ['Seattle, WA', 'Skokie, IL', 'New York City'],
+          answer: 2
+        }
+      ]
+  }
+
+  onidle () {
+    const
+      previous = this.select
+        `footer button:first-of-type`
+
+    , next = this.select
+        `footer button:last-of-type`
+
+    , beginning =
+        this.context.current == 0
+
+    , end =
+        this.context.current ==
+          (this.context.questions_and_answers.length -1)
+
+    this.enable (beginning, previous)
+    this.enable (end, next)
+  }
+
+  // helpers
+  enable (predicate, element) {
+    predicate
+      ? element.disabled = true
+      : element.disabled = false
```

Comment:
it looks kind of ok visually but this style of ternary violates some well established style rules and kinda threw me for a loop reading it at first. in general only the assignment value should be inside the ternary, to avoid duplication, and splitting a ternary across multiple lines is a sign that it would be clearer as an if/else.

```
element.disabled = (predicate ? true : false)
```
is what i'd write